upup/pkg/fi/cloudup/alitasks/launchconfiguration.go

```diff
@@ -81,52 +78,59 @@ func (l *LaunchConfiguration) Find(c *fi.Context) (*LaunchConfiguration, error)
 	if err != nil {
 		return nil, fmt.Errorf("error finding ScalingConfigurations: %v", err)
 	}
-	// Don't exist ScalingConfigurations with specified  Name.
+
+	// No ScalingConfigurations with specified Name.
 	if len(configList) == 0 {
+		klog.V(2).Infof("can't found matching LaunchConfiguration: %q", *l.Name)
 		return nil, nil
 	}
 	if len(configList) > 1 {
-		klog.V(4).Infof("The number of specified ScalingConfigurations with the same name and ScalingGroupId exceeds 1, diskName:%q", *l.Name)
+		klog.V(2).Infof("found mulitiple ScalingConfigurations with the name: %q", *l.Name)
+		return nil, nil
```

Comment:
In other cloudup tasks we return an error when finding multiple resources we expect to only find one of. For example:

https://github.com/kubernetes/kops/blob/e1f7d31c823a4ad3dff2c8767c2f7c8d361738d7/upup/pkg/fi/cloudup/awstasks/autoscalinggroup.go#L219

Should we consider doing the same here, rather than assuming the first one in the list is the desired resource?

also nit: `mulitiple` -> `multiple`

Reply:
Fixed and rebased. Thank you @rifelpet